### PR TITLE
Add guide to install istio on Huawei CCE

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: content/en/docs/setup/platform-setup/huaweicloud/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Huawei Cloud
+description: Instructions to setup an Huawei Cloud kubernetes cluster for Istio.
+weight: 25
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/huaweicloud/index.md:4]: The weight should be between 20 and 25 as those are GKE and IBM. Maybe 23?

File: content/en/docs/setup/platform-setup/huaweicloud/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      selector:
+        app: istio-ingressgateway
+        istio: ingressgateway
+      sessionAffinity: None
+      type: LoadBalancer
+    EOF
+    {{< /text >}}
+
+1.  Play with Istio and try the examples provided officially.
+
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/huaweicloud/index.md:126]: Extra empty lines.

Comment (hzxuzhonghu) [reply content/en/docs/setup/platform-setup/huaweicloud/index.md:126]: thanks

File: content/en/docs/setup/platform-setup/huaweicloud/index.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+---
+title: Huawei Cloud
+description: Instructions to setup an Huawei Cloud kubernetes cluster for Istio.
```
Comment (rvennam) [inline content/en/docs/setup/platform-setup/huaweicloud/index.md:3]:
```suggestion
description: Instructions to setup an Huawei Cloud Kubernetes cluster for Istio.
```

File: content/en/docs/setup/platform-setup/huaweicloud/index.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+`Cloud Container Engine Console`, which fully supports Istio.
+
+{{< tip >}}
+Huawei offers a {{< gloss >}}managed control plane{{< /gloss >}} add-on for the Huawei Cloud Container Engine,
+which you can use instead of installing Istio manually.
+Refer to [Huawei Application Service Mesh](https://support.huaweicloud.com/istio/)
+for details and instructions.
+{{< /tip >}}
+
+Following the [Huawei Cloud Instructions](https://support.huaweicloud.com/en-us/qs-cce/cce_qs_0008.html) to prepare a cluster before manually installing Istio, proceed as follows:
```
Comment (rvennam) [inline content/en/docs/setup/platform-setup/huaweicloud/index.md:27]: Should the user follow the instructions in the link? I'm guessing not, but it's a little confusing.

File: content/en/docs/setup/platform-setup/huaweicloud/index.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+1.  Log in to the CCE console. Choose **Dashboard** > **Buy Cluster** to open the **Buy Hybrid Cluster** page. An alternative way to open that page is to choose **Resource Management** > **Clusters** in the navigation pane and click **Buy** next to **Hybrid Cluster**.
+
+1.  On the **Configure Cluster** page, configure cluster parameters.
+    In this example, a majority of parameters retain default values. After the cluster configuration is complete, click Next: **Create Node** to go to the node creation page.
+
+    {{< tip >}}
+    Istio release has some requirement for the Kubernetes version,
+    select the version according to Istio's [support policy]](/about/supported-releases#supported-releases/).
```
Comment (rvennam) [inline content/en/docs/setup/platform-setup/huaweicloud/index.md:36]:
```suggestion
    Every Istio release has a [supported list](/about/supported-releases#supported-releases/) of Kubernetes versions. Select the Kubernetes version that matches your desired Istio version.
```

File: content/en/docs/setup/platform-setup/huaweicloud/index.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+          targetPort: 15443
+        selector:
+          app: istio-ingressgateway
+          istio: ingressgateway
+        sessionAffinity: None
+        type: LoadBalancer
+      EOF
+      {{< /text >}}
+
+1.  Play with Istio and try the examples provided officially.
```
Comment (rvennam) [inline content/en/docs/setup/platform-setup/huaweicloud/index.md:125]:
```suggestion
Start playing with Istio by trying out the various [tasks](/docs/tasks).
```

File: content/en/docs/setup/platform-setup/huaweicloud/index.md (hunk 4 above)
Comment (hzxuzhonghu) [reply content/en/docs/setup/platform-setup/huaweicloud/index.md:27]: This is an instruction about how to create a k8s using huawei cce

**Comment by rvennam** [comment]
numbering issue:
image

**Review by rvennam — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
Thank you all.

**Pull request merged**